src/store/podcast/index.ts

```diff
@@ -0,0 +1,54 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { DataStatus } from 'src/common/enums'
+import { Episode, Podcast } from 'src/common/types'
+import { loadPodcast, loadEpisodesByPodcastId } from './actions'
+
+type State = {
+  dataStatus: DataStatus
+  podcast: Podcast | null
+  episodes: Episode[]
+  episodesDataStatus: DataStatus
+  totalCount: number
+}
+
+const initialState: State = {
+  dataStatus: DataStatus.IDLE,
+  podcast: null,
+  episodes: [],
+  episodesDataStatus: DataStatus.IDLE,
```

Comment:
Not sure if really need 2 dateStatus

Reply:
I think when we get to the preloader, it can be useful